app/src/main/java/com/duckduckgo/app/trackerdetection/model/TdsEntity.kt

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.app.privacy.store
+package com.duckduckgo.app.trackerdetection.model
 
-interface PrevalenceStore {
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 
-    fun findPrevalenceOf(entity: String): Double?
+@Entity(tableName = "tds_entity")
+data class TdsEntity(
```

Comment:
Why do we need this and the `Entity` interface?

Reply:
Having the `Entity` interface allows the classes that do entity lookups to avoid being tightly coupled with the TDS list and its concrete entity type.